src/Redux/Slices/MountedComponents.js

```diff
@@ -9,6 +9,8 @@ const initialState = {
 	CoinsPurchaseWindow: false,
 	CoinsTab: false,
 	EnergyTab: false,
+	Crafting: false,
+	Seen: false,
```

Comment:
Please change `Cafting` and `Seen` keys to `CraftingWindow` and `SeenWindow`.

Reply:
Done.